### PR TITLE
Uhm 4018

### PR DESCRIPTION
**Pull request by charlesdupuyrony**
### **JIRA TICKET NAME**
[OEUI-1234](https://mocklink) mock text

### **Summary**
- sample summary

## I have checked that on this Pull request

- [ ] I can sucessfully create Drug orders
- [ ] I can sucessfully create Lab orders
- [ ] I can sucessfully search for drug orders

### CODE REVIEW

File: app/js/components/orderEntry/OrderEntryPage.jsx [hunk 1]
```diff
@@ -485,9 +496,6 @@ const mapStateToProps = ({
   dateFormatReducer,
   inpatientCareSetting,
   settingEncounterTypeReducer,
-  settingEncounterRoleReducer,
-  patient,
-  note,
```
Comment (mogoodrich) [inline app/js/components/orderEntry/OrderEntryPage.jsx]: For some reason these three lines got moved to the block down below @charlesdupuyrony ... this is what was causing that error we were seeing yesterday.

File: app/js/components/orderEntry/OrderEntryPage.jsx [hunk 2]
```diff
@@ -300,34 +316,29 @@ export class OrderEntryPage extends PureComponent {
...
-              {this.props.currentOrderType.id && this.renderDraftOrder()}
-            </div>
+            <Tabs
+              selectedIndex={this.state.index} onSelect={ (i) => {
+                this.switchTabs(i); this.setState({ index: i });
+              }
+              }>
+              <TabList>
+                <Tab><FormattedMessage id="app.orders.list" defaultMessage="Orders List" description="Orders List" /></Tab>
+                <Tab><FormattedMessage id="app.orders.labs.add" defaultMessage="Add Lab Orders" description="Add Lab Orders" /></Tab>
```
Comment (mogoodrich) [inline app/js/components/orderEntry/OrderEntryPage.jsx]: Sorry, this wasn't clear from the ticket, but there needs to be an option for "Drug Orders" as well, and then we determine which tabs are available (Drug Orders, Test Orders, or both) based on a passed-in "page" attribute (see attached screenshot)... if page=laborders only show laborders tab, if page=testorders, only show Test Orders tab, if page attribute not present, show both.



I don't remember exactly how it currently is achieved, but it looks like the logic to handle this in is SelectOrderType.jsx.

**Comment by charlesdupuyrony** [comment]
I debugged that and submitted a new pull request

**Comment by dkayiwa** [comment]
@charlesdupuyrony did you see the merge conflicts on this?

**Comment by AppBankDigital** [comment]
@mogoodrich Is it resolved.

**Pull request closed without merging**